repos/system_upgrade/el7toel8/actors/reportsettargetrelease/libraries/reportsettargetrelease.py

```diff
@@ -26,8 +26,9 @@ def _report_unhandled_release():
     # TODO: set the POST group after it's created.
     target_version = api.current_actor().configuration.version.target
     commands = [
-        ['subscription-manager', 'release', '--unset'],
-        ['subscription-manager', 'release', '--set', target_version],
+        ['sh', '-c',
+         ('subscription-manager release --unset &&'
+          'subscription-manager release --set {}'.format(target_version))]
```

Comment:
I thought we needed to execute both, so I used `&&` to string the actions together. However, `&&` is a shell primitive, hence the `sh -c 'command1 && command2'`. If we need to execute just one `sh -c` becomes redundant. 

The question now is, do we want to suggest unset or set? :)

Reply:
I expect (maybe blindly) that the default/typical scenario is to not have the release locked. So I vote for the unset.

> I thought we needed to execute both, so I used && to string the actions together. However, && is a shell primitive, hence the sh -c 'command1 && command2'. If we need to execute just one sh -c becomes redundant.

I see. Yeah, I can't come up with a better solution. If @pirat89 can't as well, then I think it's ok :)

Reply:
yep. Just one of them should be executed. the execution of both of them is nonsense. This is one of cases which we discussed we will create multiple remediation instructions per report, to be able to tell that just one of cmds should be executed.  In case of systems waiting for certifications (or in case of EUS) it is expected to have locked release to a specific release. Installing the newest one, menas usually that such users will miss certificatipon they need.